libs/dal/src/repositories/notification-template/notification-template.schema.ts

```diff
@@ -120,6 +120,9 @@ notificationTemplateSchema.virtual('steps.template', {
   justOne: true,
 });
 
+notificationTemplateSchema.path('steps').schema.set('toJSON', { virtuals: true });
+notificationTemplateSchema.path('steps').schema.set('toObject', { virtuals: true });
```

Comment:
This one is only required for nested virtual right? Because we have more virtual without nesting, but looks that they do pass as usual.

Reply:
yes. And for other virtuals, there's already a default schema option with these two enabled:
https://github.com/novuhq/novu/blob/main/libs/dal/src/repositories/schema-default.options.ts#L3